grouper-ui/java/src/edu/internet2/middleware/grouper/grouperUi/serviceLogic/UiV2Group.java

```diff
@@ -4203,5 +4204,51 @@ public void groupEditCompositeSubmit(HttpServletRequest request, HttpServletResp
       GrouperSession.stopQuietly(grouperSession);
     }
   }
+  
+  /**
+   * update a loader group

```

Comment:
Could you describe a bit more, what this method actually is doing? what exceptions may be thrown and when, etc?